modules/client/src/internal/utils.ts

```diff
@@ -76,18 +76,15 @@ export function toDaoDetails(
     plugins: dao.plugins.map(
       (
         plugin: SubgraphPluginListItem,
-      ): InstalledPluginListItem[] => {
-        return plugin.installations.map((installation) => {
-          return {
-            instanceAddress: plugin.id,
-            id:
-              `${installation.appliedVersion.pluginRepo.subdomain}.plugin.dao.eth`,
-            release: installation.appliedVersion.release.release,
-            build: installation.appliedVersion.build,
-          };
-        });
-      },
-    ).flat(),
```

Comment:
Does the `flat()` removal change any of the current results returned, regarding DAO's installing the same exact plugin instance twice?

Reply:
nope, because befor que had an array os intallations, inside each plugin, if you remember the full one plugin being installed multiple times dilema, now we only have installations so the flat does not make sense anymore